packages/wp-dashboard/src/components/editorSettings/shopping/test/shopping.js

```diff
@@ -182,4 +205,31 @@ describe('Editor Settings: Shopping <Shopping />', function () {
       shopifyHost: 'mynewstore.myshopify.com',
     });
   });
+
+  it('should render test api connection button', async function () {
+    renderWithProviders(
+      <ConfigProvider config={config}>
+        <ShoppingSettings
+          shoppingProvider={SHOPPING_PROVIDER_TYPE.SHOPIFY}
+          shopifyHost="yourstore.myshopify.com"
+          shopifyAccessToken="123"
+          updateSettings={updateSettings}
+          vendors={vendors}
+          woocommerce={woocommerceDefault}
+        />
+      </ConfigProvider>
+    );
+
+    const input = screen.getByTestId('shopify-test-connection');
+    expect(input).toBeInTheDocument();
+
+    // eslint-disable-next-line testing-library/no-unnecessary-act, require-await
+    await act(async () => {
+      fireEvent.click(input);
+    });
```

Comment:
If the linter says the `act()` is unnecessary — have you tried not adding it? 🙂 

All Testing Library utils are already wrapped in act. Most of the time, if you're seeing an act warning, it's not just something to be silenced, but it's actually telling you that something unexpected is happening in your test.

```suggestion
    fireEvent.click(input);
```

Reply:
Yes -- tried not adding ... was throwing errors about updating state.